pom.xml

```diff
@@ -2074,6 +2074,9 @@
                                 <include>*/README.md</include>
                                 <include>docker/playground/README.md</include>
                             </includes>
+                            <excludes>
+                                <exclude>docs/*/lib/python*/**/*.md</exclude>
+                            </excludes>
```

Comment:
what will happen if both `includes` and `excludes` is configured? cc @bowenliang123

Reply:
https://github.com/diffplug/spotless/blob/maven/2.30.0/plugin-maven/src/main/java/com/diffplug/spotless/maven/AbstractSpotlessMojo.java#L284